py/desisurvey/nightops.py

```diff
@@ -169,10 +172,10 @@ def nightOps(day_stats, obsplan, w, ocnt, tilesObserved, tableOutput=True, use_j
                 else:
                     # Try another target?
                     # Observe longer split into modulo(max_len)
-                    mjd += 0.25/24.0
+                    mjd += LSTres
```

Comment:
Does this change mean that it wasn't doing the right thing before?  (since 0.25 / 24 != 1 / 144)